src/components/Layout.tsx

```diff
@@ -10,28 +10,28 @@ export default function Layout({ children }: LayoutProps) {
     return (
         <div className="bg-[url('/fire-bg.jpg')]">
             <div className="bg-[url('/fire-sil.png')] bg-right-bottom bg-no-repeat bg-fixed bg-[length:310px_233px] flex flex-col">
-                <header className="header">
+                <header className="flex justify-between h-[100px] shadow-[0_5px_black]">
```

Comment:
The 100px height is a thing that does not only appear here but also in the calculation of the layout's main content. Therefore it would be cleaner to also include it as a custom width in the Tailwind config.